addons_engine/addons-manager/services/addons_runner.py

```diff
@@ -0,0 +1,291 @@
+import logging
+import os
+import threading
+
+import docker
+import requests
+from db import addons_db
+
+addons_manager = None
+
+DEFAULT_PROJECT_NAME = os.environ.get("DEFAULT_PROJECT_NAME") or "root_orchestrator"
+DEFAULT_NETWORK_NAME = f"{DEFAULT_PROJECT_NAME}_default"
+
+ADDON_ID_LABEL = os.environ.get("ADDON_ID_LABEL") or "oak.addon.id"
+ADDON_MANAGER_LABEL = os.environ.get("ADDON_MANAGER_KEY") or "oak.plugin.manager.id"
+ADDON_SERVICE_NAME_LABEL = os.environ.get("ADDON_SERVICE_NAME_LABEL") or "oak.service.name"
+
+MARKETPLACE_API = f"{os.environ.get('MARKETPLACE_ADDR')}/api/v1/marketplace"
+
+
+class DockerAddonRunner:
```

Comment:
Do you have some exemplary SLA/(test/dummy)addon I could try out to see if this works as intended?

Reply:
Good point! I'll add some steps in the documentation for it.